backend/main.go

```diff
@@ -40,6 +38,9 @@ const (
 	DATE_ORDERED_COLUMN   = 0
 	ASIN_COLUMN           = 4
 	ORIGINAL_PRICE_COLUMN = 12
+	CAPTCHA_INDICATOR     = `For information about migrating to our APIs refer to our Marketplace APIs at https://developer.amazonservices.com/ref=rm_c_sv, or our Product Advertising API at https://affiliate-program.amazon.com/gp/advertising/api/detail/main.html/ref=rm_c_ac for advertising use cases.`
+	UNAVAILABLE_INDICATOR = `<span class="a-color-price a-text-bold">Currently unavailable.</span>`
+	PRICE_INDICATOR       = `<span class="a-offscreen">$`
 )
 
 // Handler is Lambda function handler

```

Comment:
Lines 50-53 should still be simplified 

`goodVariableName, err := newItemCategories(request)
if err != nil {
  logerror
  return emptyobject, err
}

return goodVariableName, nil`